logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/ProteusClientProvider.kt

```diff
@@ -0,0 +1,38 @@
+package com.wire.kalium.logic.feature
+
+import com.wire.kalium.cryptography.ProteusClient
+import com.wire.kalium.cryptography.ProteusClientImpl
+import com.wire.kalium.cryptography.exceptions.ProteusException
+
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import kotlin.coroutines.cancellation.CancellationException
+
+interface ProteusClientProvider {
+    suspend fun clear()
```

Comment:
Why not also use the name `clearLocalFiles()` here just `clear` is a bit ambiguous about what it does.